core/common/test/UtcOffsetTest.kt

```diff
@@ -26,6 +26,11 @@ class UtcOffsetTest {
             "-01_00", "-01;00", "-01@00", "-01:AA",
             "@01:00")
 
+        val invalidStandaloneOffsets = listOf(
```

Comment:
What does "standalone" means here?

Reply:
Offsets that are invalid on their own but can be used in timezone identifiers.

Reply:
Ok, not quite clear from the name, I'd rather merge with `fixedOffsetTimeZoneIds`